rfcs/2022-01-07-simplex-services.md

```diff
@@ -0,0 +1,37 @@
+# SimpleX Services
+
+## Background and motivation
+
+SimpleX clients communicating with SimpleX services require message format separate from SimpleX Chat `x` namespace.
+
+## Proposal
+
+Use `s` (for "service") namespace with REST like messages for requests and responses, e.g.
+
+```
+s.post resource data
+s.get resource
```

Comment:
we can use parameters part of the message and have more than one data blocks already.

So the message would have the same raw format